.github/workflows/ci.yml

```diff
@@ -50,11 +54,18 @@ jobs:
           # See https://github.com/microsoft/rushstack/issues/2981
           BROWSERSLIST_IGNORE_OLD_DATA: 1
 
+          # Workaround for Node 18 incompatibility with Webpack 4
+          NODE_OPTIONS: --openssl-legacy-provider
```

Comment:
We should only set this if it's specifically necessary.

Reply:
Fixed